Zend/zend_inheritance.c

```diff
@@ -719,7 +719,7 @@ static ZEND_COLD zend_string *zend_get_function_declaration(const zend_function
 							}
 							smart_str_appendc(&str, '\'');
 						} else if (Z_TYPE_P(zv) == IS_ARRAY) {
-							smart_str_appends(&str, "Array");
+							smart_str_appends(&str, "<array>");
```

Comment:
`array(...)` or `[...]` would be options as well.

Reply:
Possibly with a distinction for empty arrays `[]`, which are likely the 95% case, which we may as well print accurately.

Reply:
Yep, these make more sense than `<array>`! So in this case, I'd vote for going with `array(...)` and `array()` because this is what we use everywhere. Are you okey with this choice?

Reply:
Oh, not exactly... The stubs use `[]` so then it's all decided. :)